04_HowTos/QueryClient/How_to_use_the_Data_Lab_QueryClient.ipynb

```diff
@@ -99,54 +120,57 @@
   },
   {
    "cell_type": "code",
    "execution_count": 3,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'ajacques'"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "## Get your Data Lab security token\n",
-    "#\n",
-    "#token = ac.login(input(\"Enter user name: (+ENTER) \"),getpass(\"Enter password: (+ENTER) \"))\n",
-    "#if not ac.isValidToken(token):\n",
-    "#   raise Exception('Token is not valid. Please check your usename/password and execute this cell again.')"
+    "#ac.login(input(\"Enter user name: (+ENTER) \"),getpass(\"Enter password: (+ENTER) \"))\n",
+    "ac.whoAmI()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### The *queryClient* class\n",
+    "<a class=\"anchor\" id=\"review\"></a>\n",
+    "# Review: General template for a simple query in SQL\n",
     "\n",
-    "All queries are executed through the <i>query()</i> method of the <i>queryClient</i> class. This takes as arguments:\n",
-    "\n",
-    "| Argument | Description | Default  value | Allowed Values |\n",
-    "|----------|-------------|----------------|----------------|\n",
-    "| adql | The query to be submitted to the TAP service | None | |\n",
-    "| sql | The query to be submitted to the DB directly | None | |\n",
-    "| fmt | The requested format (if any) | ascii | ascii,csv,votable,fits |\n",
-    "| out | The saved location (if any) | None | local filename, *vos://filename*, *mydb://tablename* |\n",
-    "| async_ | Indicates if the query is asynchronous | False | |\n",
-    "\n",
-    "All arguments are optional, except that one of *adql* or *sql* must be supplied.  The distinction between these two parameters is in how the *QueryClient* executes the query:  If *adql* is provided the query is sent to the TAP (Table Access Protocol) service, if *sql* is provided the query is sent directly to the database.  The choice of execution depends on whether the query string contains ADQL-specific functions, or SQL constructs or DB extensions not understood by the TAP service.  For large queries there can also be a performance difference depending on the where/how the results are saved. "
+    "### SQL is a way to describe to a database what you want from it\n",
+    "General template for a simple query written in SQL\n",
+    "```\n",
+    "SELECT something\n",
+    "FROM database.table\n",
+    "WHERE constraints\n",
+    "LIMIT 100\n",
+    "```\n",
+    "Please see our intro notebook *01_JupyterPythonSQL101* for more info on this."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "<a class=\"anchor\" id=\"query\"></a>\n",
-    "### A quick query\n",
+    "# A quick query\n",
     "\n",
     "Let's say we want to return the $gri$ magnitudes of the top 10 objects in the SMASH DR1 dataset and get it back as a CSV file:"
```

Comment:
Because the order of rows in the DB table is arbitrary, suggest to replace with "Let's say we want to fetch the g,r,i magnitudes from 10 objects in the SMASH DR1 data set, and retrieve the results as a CSV-formatted string."